torchgeo/datasets/cdl.py

```diff
@@ -58,6 +58,8 @@ class CDL(RasterDataset):
         (2008, "e6aa3967e379b98fd30c26abe9696053"),
     ]
 
+    cmap: Dict[int, Tuple[int, int, int, int]] = {}
```

Comment:
This is already defined at the `RasterDataset` level, I don't think we need it here too